qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/upgrades/LogsIndexModeFullClusterRestartIT.java

```diff
@@ -37,14 +37,14 @@ public class LogsIndexModeFullClusterRestartIT extends ParameterizedFullClusterR
     @ClassRule
     public static final ElasticsearchCluster cluster = ElasticsearchCluster.local()
         .distribution(DistributionType.DEFAULT)
+        .version(getOldClusterTestVersion())
         .module("constant-keyword")
         .module("data-streams")
         .module("mapper-extras")
         .module("x-pack-aggregate-metric")
         .module("x-pack-stack")
         .setting("xpack.security.enabled", "false")
         .setting("xpack.license.self_generated.type", "trial")
-        .setting("cluster.logsdb.enabled", "true")
```

Comment:
We don't need this because we explicitly set index mode in the template. And this setting does not exist in old clusters.